docs/source/support/troubleshooting/index.rst

```diff
@@ -0,0 +1,8 @@
+==========================================================
+Troubleshooting
+==========================================================
+
+.. toctree::
+   :maxdepth: 2
+
+   installation
```

Comment:
![image](https://github.com/scylladb/scylla-operator/assets/32443348/d16e4608-082a-4200-9065-e6a5300a7f2c)
Additional headers render here.

Reply:
this has `:maxdepth: 2` which intends for 1 subitem

Reply:
Ok, just wanted to make sure this was intentional